@utahdts/utah-design-system/react/components/forms/radioButton/RadioButton.jsx

```diff
@@ -0,0 +1,71 @@
+// @ts-check
+import PropTypes from 'prop-types';
+import { useCallback } from 'react';
+import { useRadioButtonsContext } from './hooks/useRadioButtonsContext';
+import RefShape from '../../../propTypesShapes/RefShape';
+
+const propTypes = {
+  className: PropTypes.string,
+  id: PropTypes.string.isRequired,
+  innerRef: RefShape,
+  isDisabled: PropTypes.bool,
+  label: PropTypes.string.isRequired,
+  name: PropTypes.string,
+  value: PropTypes.string,
+};
+const defaultProps = {
+  className: null,
+  isDisabled: false,
+  innerRef: null,
+  name: null,
+  value: null,
+};
+
+/**
+ * @param {Object} props
```

Comment:
[props.className] means the value can also be `undefined`. This is where PropTypes and JsDoc differ slightly in that the PropTypes have a default of null, but the JSDoc says the default is undefined. 
What do we think about taking the ` | null` off of all the props and checking for `undefined` instead of `null` in the component? `undefined` means it wasn't set, which makes sense for parameters that weren't passed in. `null` mean it was set but doesn't have a value which isn't quite true...